tests/llama_index/conftest.py

```diff
@@ -72,7 +72,7 @@ def settings(monkeypatch, mock_openai):
     )
     monkeypatch.setattr(Settings, "llm", OpenAI())
     monkeypatch.setattr(Settings, "embed_model", OpenAIEmbedding())
-    monkeypatch.setattr(Settings, "callback_manager", CallbackManager([TokenCountingHandler()]))
+    monkeypatch.setattr(Settings, "callback_manager", CallbackManager([LlamaDebugHandler()]))
```

Comment:
What is this change for?

Reply:
This is for the issue 1, the `TokenCountingHandler` has an issue with LlamaIndex 0.10.57, so we need to use a different one as an example🙂